zppy/bundle.py

```diff
@@ -82,22 +80,23 @@ def add_task(self, scriptFile, dependFiles):
     def display_dependencies(self):
         print(f"Displaying dependencies for {self.bundle_name}")
         print("dependencies_internal:")
-        # Note to Ryan: I don't think te for else structure is working as expected
```

Comment:
Possible conditions:
1. List in `for` block is empty.
2. `for` block completes without hitting `break`.
3. `for` block ends early because of `break`.

The `else` block runs in (1) and (2) but not in (3). It looks like I got confused here and thought the `else` block only ran in (1). That's why it worked in the `handle_bundles` `for-else` block; there's a `break`.